src/Categories/NaturalTransformation/NaturalIsomorphism.agda

```diff
@@ -71,6 +71,26 @@ record NaturalIsomorphism {C : Category o ℓ e}
       }
     }
 
+-- Like IsIso for morphisms, a natural transformation may be an isomorphism
+-- if there exists some inverse to it.
+-- To be consistent, there's also an IsNI for natural isomorphisms
+record IsNI {C : Category o ℓ e}
+            {D : Category o′ ℓ′ e′}
+            {F G : Functor C D}
+            (F⇒G : NaturalTransformation F G) : Set (o ⊔ ℓ ⊔ ℓ′ ⊔ e′) where
+  field
+    F⇐G : NaturalTransformation G F
+
```

Comment:
You can now delete this blank line and extra `field`

Reply:
Sorry for the misstyling ><

Reply:
No problem! The time to review is small compared to the effort of doing PRs.